aws/resource_aws_elasticsearch_domain.go

```diff
@@ -151,6 +152,22 @@ func resourceAwsElasticSearchDomain() *schema.Resource {
 								elasticsearch.TLSSecurityPolicyPolicyMinTls12201907,
 							}, false),
 						},
+						"custom_endpoint_enabled": {
+							Type:     schema.TypeBool,
+							Optional: true,
+							Default:  false,
+						},
+						"custom_endpoint": {
+							Type:     schema.TypeString,
+							Optional: true,
+							DiffSuppressFunc: isCustomEndpointDisabled,
```

Comment:
This will appease the linters: 😄 

```suggestion
							Type:             schema.TypeString,
							Optional:         true,
							DiffSuppressFunc: isCustomEndpointDisabled,
```